runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerConfig.java

```diff
@@ -54,4 +54,24 @@ default boolean rememberTileColors()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showMinimap",
+		name = "Show on minimap",
+		description = "Shows marked tiles on the minimap"
+	)
+	default boolean showMinimap()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+			keyName = "minimapOpacity",
+			name = "Minimap Overlay Opacity",
+			description = "The opacity of the minimap markers [0, 255]"
+	)
+	default int minimapOverlayOpacity()
```

Comment:
Is this really required? I would just pick something which looks good and use that.